class-sc-wp-updates-notifier.php

```diff
@@ -259,6 +266,154 @@ public function plugin_action_links( $links, $file ) {
+
+				.column-update_notifications{
+					width: 15%;
+				}
+
+				.sc_wpun_btn:before {
+					font-family: "dashicons";
+					display: inline-block;
+					-webkit-font-smoothing: antialiased;
+					font: normal 20px/1;
+					vertical-align: top;
+					margin-right: 5px;
+					margin-right: 0.5rem;
+				}
+
+				.sc_wpun_btn_enable:before {
+					content: "\f12a";
+					color: green;
+				}
+
+				.sc_wpun_btn_disable:before {
+					content: "\f153";
+					color: red;
+				}
+
+				.sc_wpun_btn_enable:hover:before {
+					content: "\f153";
+					color: red;
+				}
+
+				.sc_wpun_btn_disable:hover:before {
+					content: "\f12a";
+					color: green;
+				}
+
+				</style>';
+			}
+		}
+
+
+		/**
+		 * Custom js for the plugins.php page.
+		 */
+		public function custom_admin_js() {
+			?>
+			<script type="text/javascript" >
+			jQuery(document).ready(function($) {
+				$( '.sc_wpun_btn' ).click(function(e) {
+					e.preventDefault();
+
+					var data = {
+						'action': 'toggle_plugin_notification',
+						'toggle': $(e.target).data().toggle,
+						'plugin_file': $(e.target).data().file,
+						'_wpnonce': "<?php echo esc_attr( wp_create_nonce( 'toggle_plugin_notification' ) ); ?>",
+					};
+
+					jQuery.post(ajaxurl, data, function(response) {
+						console.log(response);
+						if ( 'success' == response ) {
+							if ( 'disable' == $(e.target).data().toggle ) {
+								$(e.target).data( 'toggle', 'enable' );
+								$(e.target).removeClass( 'sc_wpun_btn_enable' );
+								$(e.target).addClass( 'sc_wpun_btn_disable' );
+								$(e.target).text( '<?php echo esc_html( __( 'Notifications Disabled', 'wp-updates-notifier' ) ); ?>' );
+							} else {
+								$(e.target).data( 'toggle', 'disable' );
+								$(e.target).removeClass( 'sc_wpun_btn_disable' );
+								$(e.target).addClass( 'sc_wpun_btn_enable' );
+								$(e.target).text( '<?php echo esc_html( __( 'Notifications Enabled', 'wp-updates-notifier' ) ); ?>' );
+							}
+						}
+					});
+				});
+			});
+			</script>
+			<?php
+		}
+
+
+		public function toggle_plugin_notification() {
+			check_ajax_referer( 'toggle_plugin_notification' );
+			if ( isset( $_POST['plugin_file'] ) && isset( $_POST['toggle'] ) && current_user_can( 'update_plugins' ) && current_user_can( 'manage_options' ) ) {
+				$plugin_file = sanitize_text_field( wp_unslash( $_POST['plugin_file'] ) );
+				$toggle      = sanitize_text_field( wp_unslash( $_POST['toggle'] ) );
+
+				$options        = $this->get_set_options( self::OPT_FIELD ); // get settings
+				$active_plugins = array_flip( get_option( 'active_plugins' ) );
+
+				if ( 'disable' === $toggle ) {
+					$options['disabled_plugins'][ $plugin_file ] = 1;
+					echo 'success';
+				} elseif ( 'enable' === $toggle ) {
+					unset( $options['disabled_plugins'][ $plugin_file ] );
+					echo 'success';
+				} else {
+					echo 'failure';
+				}
+				$output = $this->get_set_options( self::OPT_FIELD, $options ); // update settings
+			}
+			wp_die();
```

Comment:
Just use die() here, as wp_die() will also print a message.  Not necessary to use here, but FYI there are functions wp_send_json() and wp_send_json_error() which can be handy!